docs/configuration/features/input_features.md

```diff
@@ -58,6 +58,33 @@ Each input feature can specify its own preprocessing via the `preprocessing` sub
 It's also possible to specify preprocessing rules for all features of a certain type. See
 [Type-Global Preprocessing](../defaults.md#type-global-preprocessing).
 
+## Common Parameters
+
+### Missing Value Strategy
+
+Ludwig allows any input feature to be "missing" in the dataset (at both training and prediction time). How
+missing values are handled by default varies depending on the feature type, but this handling strategy can be configured
+in `preprocessing` section of the config for each feature:
+
+```yaml
+preprocessing:
+    missing_value_strategy: fill_with_mean
+```
+
+Options:
+
+- **`fill_with_const`**: Replaces the missing value with a specific value specified with the `fill_value` parameter.
+- **`fill_with_mean`**: Replaces the missing values with the mean of the values in the column.
+- **`fill_with_mode`**: Replaces the missing values with the most frequent value in the column.
+- **`fill_with_false`** Replace the missing values with the `false` value in the column (`binary` features only).
+- **`bfill`**: Replaces the missing values with the next valid value from the subsequent rows of the input dataset.
+- **`ffill`**: Replaces the missing values with the previous valid value from the preceding rows of the input dataset.
+- **`drop_row`**: Removes the entire row from the dataset if this column is missing.
```

Comment:
Wondering if we can populate these from the schema as well

Reply:
Also, `fill_with_mean` is not supported for category features, only number features

Reply:
Not at the moment, since it's not in the schema as such, but we should work to getting there.

Reply:
Sounds good, then it would be good if we can just mention that `fill_with_mean` is only support for number features

Reply:
Good call!